caravel/assets/javascripts/SqlLab/components/TableElement.jsx

```diff
@@ -30,66 +32,85 @@ class TableElement extends React.Component {
     };
     this.props.actions.addQueryEditor(qe);
   }
+
+  collapseTable(e) {
+    e.preventDefault();
+    this.props.actions.collapseTable.bind(this, this.props.table)();
+  }
+
+  expandTable(e) {
+    e.preventDefault();
+    this.props.actions.expandTable.bind(this, this.props.table)();
+  }
+
   render() {
     let metadata = null;
     let buttonToggle;
     if (this.props.table.expanded) {
       buttonToggle = (
-        <Link
+        <a
           href="#"
-          onClick={this.props.actions.collapseTable.bind(this, this.props.table)}
-          placement="right"
-          tooltip="Collapse the table's structure information"
+          onClick={(e) => { this.collapseTable(e); }}
         >
-          {this.props.table.name} <i className="fa fa-caret-up" />
-        </Link>
+          <strong>{this.props.table.name}</strong>
+          <small className="m-l-5"><i className="fa fa-minus" /></small>
+        </a>
       );
       metadata = (
         <div>
           {this.props.table.columns.map((col) => (
-            <div className="clearfix">
-              <span className="pull-left m-l-5">{col.name}</span>
-              <span className="pull-right text-muted">{col.type}</span>
+            <div className="row">
+              <div className="col-sm-8">
```

Comment:
discussed and the browser size that using the grid here would pose a problem at is basically tablet size, and we haven't designed for the breakpoint anywhere else yet.